website/public/locales/fa/side_menu.json

```diff
@@ -0,0 +1,12 @@
+{
+  "dashboard": "داشبورد",
```

Comment:
We had some refactoring in another PR. Sorry for another ask but can you move all these entries in to common.json?